### PR TITLE
Updated the Wiki Token page by adding a detailed description of features found on the Token menu

### PR DESCRIPTION
**Pull request by Jonyango**
I was unsure of the format to follow while adding the detailed description of the features so I decided to use the format above.

### CODE REVIEW

File: wiki/Tokens.md [hunk 1]
```diff
@@ -1,9 +1,26 @@
...
+
+` visible to players `  hides/show the token to players.<br>
+
+` moveable by players ` allows the players to move the token from one location to another.<br>
+
+` lock `  locks a token in place making it immovable. **Locking a token makes it impossible for both the Dungeon master and the players to move or delete the token.**
+
+` delete `  removes a token from the map.
 
 ![token-menu](https://user-images.githubusercontent.com/9096667/87476985-f3b68400-c5ec-11ea-8165-21d0f2c150d1.png)
```
Comment (maxb2) [inline wiki/Tokens.md:20]: I think this is supposed to be the image from here: https://github.com/dungeon-revealer/dungeon-revealer/issues/768#issuecomment-702433295

File: wiki/Tokens.md [hunk 2]
```diff
@@ -1,9 +1,26 @@
+### About
 Tokens are shapes that can be added to the map representing things such as creatures, players, items, area effects, notes or anything else you can think of.
+### Usage
+To add a token, click on the token tool on the left control panel and then click anywhere on the map to place the token. The token size will default to the grid size if it is present. <br> To customize a token,right click on a token to reveal the token menu. From the token menu you can name the token, change its size and color, hide/show the token to players, and link a note.
+### Description of Features on the Token Menu
+` Label `  this is the token identifier.It can be set to any value either a string or a number with the default value being 1.<br>
```
Comment (maxb2) [inline wiki/Tokens.md:6]: Missing a space after the period.

File: wiki/Tokens.md [hunk 3]
```diff
@@ -1,9 +1,26 @@
+### About
 Tokens are shapes that can be added to the map representing things such as creatures, players, items, area effects, notes or anything else you can think of.
+### Usage
+To add a token, click on the token tool on the left control panel and then click anywhere on the map to place the token. The token size will default to the grid size if it is present. <br> To customize a token,right click on a token to reveal the token menu. From the token menu you can name the token, change its size and color, hide/show the token to players, and link a note.
```
Comment (maxb2) [inline wiki/Tokens.md:4]: Missing a space after the comma.

**Comment by maxb2** [comment]
This looks great! I'm happy with this once you've resolved everything.

**Comment by Jonyango** [comment]
@maxb2 I have made the suggested edits. Do check it out and let me know. Thank you.

**Comment by maxb2** [comment]
Merged. Closes #768. Thanks for the contribution!

**Pull request merged**